src/main/java/com/xpandit/plugins/xrayjenkins/Utils/BuilderUtils.java

```diff
@@ -31,6 +32,23 @@ public static boolean isSupportedJobType(Class<? extends AbstractProject> jobTyp
                 || MavenModuleSet.class.isAssignableFrom(jobType);
     }
 
+    /**
+     * Utility method to check if the endpoint supports glob expressions
+     * @param endpointValue the endpoint value
+     * @return <code>true</code> if the endpoint supports glob expressions , <code>false </code> otherwise
+     */
+    public static boolean areGlobExpressionsSupported(Endpoint endpointValue) {
```

Comment:
boolean returning methods should always start with "is" -> _isGlobExpressionsSupported_
If there were several arguments, you could called it: _isAnyGlob..._